src/app/core/services/crypto-info.service.ts

```diff
@@ -46,6 +46,13 @@ export class CryptoInfoService {
       `https://api.coingecko.com/api/v3/coins/markets?vs_currency=${currency}&ids=${cryptoId}&order=market_cap_desc&per_page=100&page=1&sparkline=false`
     );
   }
+getCryptoList(){
+  return this.http.get(
+    `https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&page=1&sparkline=true&price_change_percentage=1h%2C7d&locale=en`)
+}
+getGlobalMarketCap(){
+  return this.http.get('https://api.coingecko.com/api/v3/global');
+}
```

Comment:
Using Internal API: As per the guidelines, it's preferred to use our internal API (https://api2.satt-token.com/getPrices) for fetching price data. This will help us maintain data consistency and minimize reliance on external APIs. 

Error Handling: The current code seems to lack error handling that might occur during the API call. It would be beneficial to add error handling in both methods to avoid potential silent failures.

Refactoring: The API URLs are hard-coded in the code, which can cause issues if they change in the future. It would be good to move them to a configuration or environment file.